docs/docs/guides/migrate/sources/auth0-guide.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Migrating Users from Auth0 to ZITADEL (Including Password Hashes)
+sidebar_label: Auth0 Migration Guide
+---
+
+## 1. Introduction
+
+**Why migrate from Auth0 to ZITADEL?**  
```

Comment:
Having this question here feels a little weird since we are not answering it. 

We should either have an answer to this our link to some resources 😁